packages/strapi-helper-plugin/lib/src/components/InputToggle/styles.scss

```diff
@@ -1,13 +1,13 @@
 .gradientOff {
   background-image: linear-gradient( to bottom right, #F65A1D, #F68E0E );
   color: white !important;
-  z-index: 0!important;
-  
+  z-index: 0 !important;
```

Comment:
@websocket98765 thanks a lot for contributing however this file no longer exists. Do you mind updating your PR with the new codebase so we can review it?